src/cardproduct.css

```diff
@@ -0,0 +1,101 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.card-container {
+  display: flex;
+  font-style: normal;
+}
+
+.card-product {
+  padding: 0;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 198px;
+  height: 406px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card-product__image {
+  height: 134px;
+  width: 160px;
+  margin-top: 32px;
+  margin-left: 19px;
+}
+
+.card-product__image_size {
+  height: 134px;
+  width: 160px;
+}
+
+.card-product__name {
+  width: 166px;
+  height: 36px;
+  margin-top: 40px;
+  margin-left: 16px;
```

Comment:
you have margin-left here and in other places. but you can just add `padding: 0 16px;` to `.card-product` selector and don't add this style everywhere.
try it:)